new_django_project/blog/api/models.py

```diff
@@ -9,3 +9,12 @@ class Post(models.Model):
     class Meta:
         ordering = ['created']
 
+class Comment(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    body = models.TextField(blank=False)
+    owner = models.ForeignKey('auth.User', related_name='comments', on_delete=models.CASCADE)
+    post = models.ForeignKey('Post', related_name='comments', on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ['created']
```

Comment:
There is other useful setting that we can use here like `indexes` to create indexes in the table that will help us to fetch the data faster. As the `comments` table tend to have a large amount of data, adding indexes seems like a good idea to have a good performance. Have you though about adding indexes in this table? (or are we going to add those on another PR?)

Reply:
That's a great idea! But... no, I haven't really thought about that. but now it's been added to the backlog 😅